kplc

```diff
@@ -18,7 +19,12 @@ fatal() { notif "[FATAL] $@" ; exit 1 ; }
 
 notif_exec() {
     notif "$@"
-    "$@"
+
+    if ${result_only}; then
+        "$@" | tr '\n' ' ' | sed "s/.*<k>\(.*\)~> \. .*/\1/g"
+    else
+        "$@"
+    fi
```

Comment:
This should probably not be done in `notif_exec` function, becase it will be used eventually for calling things that are not `krun` (and is already used for `run_kompile`. So I would inline this logic directly into `run_interpreter`.